docs/browser-lifecycle.md

```diff
@@ -0,0 +1,74 @@
+| `browser.page_load` | Summarizes the user-visible page load experience from navigation start until LCP (includes total duration, paint timings, resource counts, transfer size). Occurs once per page load. | Not created | Not created                                                                                                                                          |
+| `browser.performance_resource_timing` | Captures information about individual resources loaded during the initial page load, from the [PerformanceResourceTiming](https://developer.mozilla.org/docs/Web/API/PerformanceResourceTiming) API. May occur multiple times per page load. | Not created | Merged (similar) [instrumentation-document-load](https://github.com/open-telemetry/opentelemetry-js-contrib/tree/main/packages/instrumentation-document-load) |
+
+---
+
+### User Interaction Phase
+
+**Starts after LCP → Ends when the page begins unloading**  
+Captures user-centric metrics, runtime behavior, and late resource loading.
+
+**All events in this phase may occur multiple times.**
+
+| Event | Description | Semantic Conventions Status | Instrumentation Status                                                                                                                                        |
+|---|---|---|---------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `browser.page_view` | Represents a **soft navigation** (SPA virtual page change). | In review [PR1910](https://github.com/open-telemetry/semantic-conventions/pull/1910) | In review [PR2386](https://github.com/open-telemetry/opentelemetry-js-contrib/pull/2386)                                                                      |
+| `browser.web_vital` | Captures Web Vitals metrics such as CLS, INP, and FID that occur after page load. | Merged [WebVitals](https://opentelemetry.io/docs/specs/semconv/browser/events/#webvital-event) | Not created                                                                                                                                                   |
+| `browser.error` | Captures unhandled JavaScript errors. | Not created | Not created                                                                                                                                                   |
+| `browser.console` | Captures browser console messages such as warnings and logs. | Not created | Not created                                                                                                                                                   |
+| `browser.user_interaction` | Captures user input events (clicks, scrolls, keypresses) and their latency. | Not created | Not created                                                                                                                                                   |
```

Comment:
There is an existing PR [here](https://github.com/open-telemetry/semantic-conventions/pull/1941) (also from @Karlie-777).